src/server/src/v1/pulse/infrastructure/database/knex.ts

```diff
@@ -101,6 +93,8 @@ export const knexConfig: KnexTypes.Config = {
   // },
 }
 
+export const knex = Knex(knexConfig)
```

Comment:
instead of exporting the config, exporting knex for if we end up making the knex configuration more environment aware